src/core/promgr/pmsdriver/local/local.go

```diff
@@ -132,23 +132,16 @@ func (d *driver) Update(projectIDOrName interface{},
 func (d *driver) List(query *models.ProjectQueryParam) (*models.ProjectQueryResult, error) {
 	var total int64
 	var projects []*models.Project
-	var groupDNCondition string
+	var gpi group.PrivilegeInterface
+	gpi = &group.DefaultGroupPrivilege{}
```

Comment:
Considering `CreatePrivilegeInterface` is already handling the fallback case, we don't need the defaultvalue here. so that the `DefaultGroupPrivilege` will not need to be exposed from the package?